stdlib/mysql/src/main/java/org/ballerinalang/mysql/Utils.java

```diff
@@ -0,0 +1,87 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.mysql;
+
+import org.ballerinalang.jvm.values.DecimalValue;
+import org.ballerinalang.jvm.values.MapValue;
+import org.ballerinalang.jvm.values.MapValueImpl;
+
+/**
+ * This class includes utility functions.
+ */
+public class Utils {
+
+    static MapValue generateOptionsMap(MapValue mysqlOptions) {
+        if (mysqlOptions != null) {
+            MapValue<String, Object> options = new MapValueImpl<>();
+            addSSLOptions(mysqlOptions.getMapValue(Constants.Options.SSL), options);
+
+            long connectTimeout = getTimeout(mysqlOptions.get(Constants.Options.CONNECT_TIMEOUT_SECONDS));
+            if (connectTimeout > 0) {
+                options.put(Constants.DatabaseProps.CONNECT_TIMEOUT, connectTimeout);
+            }
+
+            long socketTimeout = getTimeout(mysqlOptions.get(Constants.Options.CONNECT_TIMEOUT_SECONDS));
```

Comment:
This should be `Constants.Options.SOCKET_TIMEOUT_SECONDS`?

Reply:
Fixed